api_app/playbooks_manager/controller.py

```diff
@@ -0,0 +1,283 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+from typing import Dict, List
+
+from celery import chord, uuid
+from django.conf import settings
+from api_app.analyzers_manager.dataclasses import AnalyzerConfig
+from api_app.connectors_manager.dataclasses import ConnectorConfig
+
+from api_app.exceptions import AlreadyFailedJobException, NotRunnableAnalyzer,  \
+    NotRunnableConnector, NotRunnablePlaybook
+
+from api_app.helpers import get_now
+from ..models import Job
+from intel_owl.consts import DEFAULT_QUEUE
+
+from .dataclasses import PlaybookConfig
+
+logger = logging.getLogger(__name__)
+
+
+def filter_playbooks(serialized_data: Dict, warnings: List[str]) -> List[str]:
+    # init empty list
+    cleaned_playbook_list = []
```

Comment:
I think `valid_playbook_list`, or `to_be_run_playbook_list`, or something like that are a better name for what you are trying to do

Reply:
Alright. `valid_playbook_list` sounds better. Will use it.